collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -676,4 +654,39 @@ private boolean loadSharedPreferencesFromFile(File src) {
         return res;
     }
 
+    /*
```

Comment:
Please don't make refactoring changes in addition to feature adds or problem fixes! I know it's tempting because the code needs it but it will be very helpful if you do them as separate PRs. That makes each PR more focused, easier to review and therefore less likely to introduce accidental bugs. Thank you! 😊 

To be clear, these are good changes so I'll go ahead and merge them in.